hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FairScheduler.java

```diff
@@ -1774,4 +1774,8 @@ protected void decreaseContainer(
   public float getReservableNodesRatio() {
     return reservableNodesRatio;
   }
+
+  long getNMHeartbeatInterval() {
```

Comment:
Trivial, unambiguous @Private method that is very obvious. Don't think javadoc adds any information.